third_party/2and3/mypy_extensions.pyi

```diff
@@ -1,6 +1,24 @@
-from typing import Dict, Type, TypeVar
+from typing import Dict, Type, TypeVar, Optional, Any
 
 T = TypeVar('T')
 
 
 def TypedDict(typename: str, fields: Dict[str, Type[T]]) -> Type[dict]: ...
+
+class Arg(object):
+    def __init__(name: Optional[str]=...,
+                 typ: Type[T]=...,
+                 keyword_only: Optional[bool]=...) -> None:
+        ...
+
+class DefaultArg(object):
+    def __init__(name: Optional[str]=...,
+                 typ: Type[T]=...,
+                 keyword_only: Optional[bool]=...) -> None:
```

Comment:
Oops these actually shouldn't be optional.  What I get for reviewing my own code.